datajunction-server/datajunction_server/internal/access/authentication/github.py

```diff
@@ -42,7 +42,7 @@ def get_github_user(access_token: str) -> Optional[User]:  # pragma: no cover
     ).json()
     if "message" in user_data and user_data["message"] == "Bad credentials":
         return None
-    session = next(get_session())
+    session = next(get_session())  # type: ignore
```

Comment:
These two functions don't look like they're actually used. I'm not going to remove them here since I want to keep the changes in this PR contained